reports/blueprint/report.tex

```diff
@@ -183,16 +200,80 @@ \subsection{Validation}
+\noindent
+The block diagram above is a simplified version of the schematic and thus does
+not enumerate each part on the PCB. A full bill of materials for the PCB can be
+found in \ref{appendix:bom}. The two critical blocks of the diagram are the USB
+FIFO and ADC. Data rate and sample rate can be found in Table
+\ref{hw:specs-table} containing the hardware specifications. Voltage samples are
+fed into TeachEE via the BNC connectors on the PCB. Two additional voltage
+channels are added for redundancy purposes, but the minimum requirement is one
+channel. By adding additional functionality to the hardware, the team has room
+to make extensions to its software in the future. The ADC input going directly
+to the FPGA is filtered by a Low Pass Filter (LPF) to prevent aliasing in the
+frequency domain. The discrete ADC also has an LPF in addition to a dedicated
+driver for the differential inputs on the chip. The driver amplifier protects
+the ADC against out of bounds voltages.
+
 \subsection{Software Design}
 \subsection{Milestones and Division of Labour}
 % Table of major and intermediate milestones (ERIC)
 % Each table cell should include due date and assigned member
 \section{Budget} % Ethan
+The budget of 600 dollars provided by the department is sufficient for this
+project assuming only one unit of the device is manufactured. In reality three
+PCB units are on order. However, the group will only submit a reimbursement
+request for one unit allowing the team to stay within budget. The extra two
+units will be treated as a personal expense since group members will be able to
+keep the extra devices and only submit the one paid for by the ECE department.
+The following table breaks down the cost of the PCB fabrication and assembly as
+quoted by the supplier. The supplier contracted for TeachEE is Bittele (7PCB),
+they take care of both the fabrication and soldering of the PCBs. Moreover,
+Bittele takes the generated output from the EDA software and procures all the
+parts directly to their facility.
+
+\begin{table}[h!]
+    \caption{TeachEE Bill of Materials}
+    \begin{tabularx}{\textwidth}{l|l|l}
+        \textbf{Item / Part Number} & \textbf{Supplier} & \textbf{Cost (USD)} \\
+        \hline
+        PCB Fabrication & Bittele & \$104.37\\
+        PCB Total BOM Procurement & Bittele & \$126.67\\
+        PCB Assembly & Bittele & \$289.76\\
+        \hline
+        Total & & \$520.80\\
+        \hline
+        Budget Remaining & & \$79.20
+    \end{tabularx} 
+\label{tab:abbreviated-bom}
+\end{table}
+
+\noindent
+As shown above, a full unit of the PCB comes in \$80 under budget. This leaves
+the time with some additional budget to purchase any replacement components or
+cover unexpected expenses. Should the project still exceed the budget, the team
+will first contact the project supervisors to see if there is a possibility of
+additional funding. The fully expanded component BOM for the PCB is provided in
+Appendix \ref{appendix:bom}.
+
```

Comment:
Isn't there supposed to be two subsections here? Split this into materials/supplies and contributions from other sources

Reply:
Ignore